src/GeneralStateTestsFiller/stEIP1153/16_tloadGasFiller.yml

```diff
@@ -0,0 +1,81 @@
+# 16
+# TLOAD(x) should cost 100 gas, same as SLOAD(x)
+#
+# Expect storage slot 0 of address a to be 100, representing the gas of sload.
+# Expect storage slot 1 of address a to be 100, representing the gas of tload.
+# Expect storage slot 2 of address a to be 1, representing sload and tload have equal gas.
+
+16_tloadGas:
+  _info:
+    comment: tload costs 100 gas same as a warm sload
+
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 0x20000
+    currentNumber: 1
+    currentTimestamp: 1000
+    currentGasLimit: 0x10000000000000
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+    currentBaseFee: 10
+
+  pre:
+    000000000000000000000000000000000000000A:
+      balance: 1000000000000000000
+      nonce: 0
+      code: |
+        :yul {
+          let extraOpcodesGas := 5
+
+          // calculate warm sload gas
+          let s := sload(5)
+          let g1 := gas()
+          let s2 := sload(5)
+          let g2 := gas()
+          sstore(0, sub(sub(g1, g2), extraOpcodesGas))
```

Comment:
I don't think it's worth it to include the gas costs for `sload` here, since this is an entirely different test suite, if in the future for some reason the `sload` gas cost changes, this test will fail.

Reply:
makes sense, fixed https://github.com/ethereum/tests/pull/1091/commits/947ebcaeb3e33266f7af456722057536595db53a & https://github.com/ethereum/tests/pull/1091/commits/2556022df9340b4fe1911a64478885b26f22fb9b